spec/services/chunkers/basic_image_chunker_spec.rb

```diff
@@ -0,0 +1,5 @@
+RSpec.describe Chunkers::BasicImageChunker do
+  describe "success" do
+    pending "add some examples (or delete) #{__FILE__}"
```

Comment:
Are you planning on adding tests for this, or do you think they're not needed here since you're testing it with the `supported_chunking_method` shared examples?

Reply:
I think we will need tests here since the shared examples are for Text.